Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:xenial
+
+# Apt packages
+RUN apt-get update
+# We don't want any interactive prompts
+RUN DEBIAN_FRONTEND=non-interactive apt-get install -yq doxygen graphviz
+
+# Set WORKDIR
+WORKDIR /documentation
+
+# Copy all files from this project root host computer to /documentation in the docker container
+ADD . /documentation
```

Comment:
This creates a big dockerfile, and requires a rebuild everytime. Better to not have this and just mount the volume directly. Look at projects at CSI for example.

Reply:
ok, so I had the mounting done right. I just thought I'd have to do a ADD (copy) too haha